azavea/cac-tripplanner-android
# Feature/dpd/disable geofencing

## Overview

Removes geofencing functionality from the application.

### Notes

- This was pretty straightforward. I did some removal of permissions that might arguably bleed into #188 , but I didn't want them to fall through the cracks, and without removing them the app would unnecessarily complain if you didn't grant it background location permissions.
- I added a PR template.

## Testing Instructions

- [Set up your Android development environment](https://github.com/azavea/cac-tripplanner-android/issues/187)
- Build and run the application. If you don't get a permissions prompt on launch, clear out the location permissions for the app.
- When the app asks for permission, confirm that it doesn't show the prompt about asking for background permissions, and that if you grant permissions "only when the app is in use", the app works fine and doesn't complain that you haven't granted the proper permissions.
- Play around with the app and make sure everything continues to work as expected.
- I attempted to remove all code that became unused or superfluous as a result of removing the geofencing functionality, but I might have missed something, so if you want, dig through the code and double-check that it's all gone.

Closes #186
